packages/tools/src/utilities/scroll.ts

```diff
@@ -31,6 +33,26 @@ export default function scroll(
   }
 }
 
+/**
+ * _getDeltaFrameIndex is a subset of functionality copied from csUtils.snapFocalPointToSlice
+ * @param delta
+ * @param sliceRange
+ * @param spacingInNormalDirection
+ * @returns { int, int }
+ */
+function _getDeltaFrameIndex(delta, sliceRange, spacingInNormalDirection) {
```

Comment:
can you add the exports you want from the `getVolumeViewportScrollInfo` so that we don't recomute these again?